pkg/blackduck/ctl_blackduck.go

```diff
@@ -215,6 +217,8 @@ func (ctl *Ctl) AddSpecFlags(cmd *cobra.Command, master bool) {
 	cmd.Flags().StringVar(&ctl.AdminPassword, "admin-password", ctl.AdminPassword, "'admin' password of Postgres database")
 	cmd.Flags().StringVar(&ctl.PostgresPassword, "postgres-password", ctl.PostgresPassword, "'postgres' password of Postgres database")
 	cmd.Flags().StringVar(&ctl.UserPassword, "user-password", ctl.UserPassword, "'user' password of Postgres database")
+	cmd.Flags().StringVar(&ctl.EnableBinaryAnalysis, "enable-binary-analysis", ctl.EnableBinaryAnalysis, "If true, enable binary analysis [true|false]")
```

Comment:
Make these BoolVar.  The flag name (enable-...) details what is being done and we only want true/false.  It should  default to true.

Reply:
@rrati I thought we decided to use StringVar for all BoolVar flags because of this exact scenario. If it defaults to true then the user needs to put `--flag=false` to turn it off. `--flag false` will error because the flag doesn't take in parameters.

Reply:
What I said in the original comment is a bit unclear.  Oops.  I meant the feature should default to disabled, but when you use --enable-binary-analysis it enables the feature.  To disable after enabled you should do --enable-binary-analysys=false (or " false").

This is the pattern we should use for any flags that will enable/disable functionality imo.  Whether we do =false or " false" is semantic, but I'd lean towards lining up with kubectl/oc for consistency.  If doing that breaks the existing interface then it's not that important.  The bool path seems easier and cleaner to handle in code, but not at the expense of breaking the existing interface.

Reply:
Done